tests/e2e/specs/out-of-game/home.spec.js

```diff
@@ -97,6 +97,21 @@ describe('Home - Game List', () => {
         .contains('Game made by other player');
     });
 
+    it('Displays "Empty" when there are no players', () => {
+      const players = [];
+      cy.createGamePlayer({ gameName: 'Game with no players', isRanked: false, players });
+      cy.get('[data-cy=game-list-item]').contains('Empty');
```

Comment:
Let's additionally check that there is only one game
```suggestion
      cy.get('[data-cy=game-list-item]').should('have.length', 1).contains('Empty');
```